sources/SilkTouch/SilkTouch/Program.cs

```diff
@@ -62,46 +72,99 @@
     // Register MSBuild
     MSBuildLocator.RegisterDefaults();
 
+#pragma warning disable ST0005 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+    JobContext jobContext = new();
```

Comment:
Surely this is the raciest of race conditions given that the `jobContext.JobKey` assignment below is within a `Parallel.ForEachAsync`?

Reply:
Oops, forgot it was async local.